src/main/kotlin/FetchTxIdsCommand.kt

```diff
@@ -0,0 +1,30 @@
+import kotlinx.cli.*
+import kotlinx.cli.ExperimentalCli
+import kotlinx.coroutines.runBlocking
+
+@OptIn(ExperimentalCli::class)
+class FetchTxIdsCommand(private val apiClient: MempoolClient = MempoolClient()) :
```

Comment:
Avoid default values in constructors, you should force the caller to think about what value they give you:

```suggestion
class FetchTxIdsCommand(private val apiClient: MempoolClient) :
```

Same in the other subcommand.